### DISCUSSION TITLE
Licensing issue

### DISCUSSION THREAD

**Discussion by tonyg** [rabbitmq/rabbitmq-server#10782] [Other]
Hi team, would you mind taking a look at (and resolving) https://github.com/rabbitmq/rabbitmq-udp-exchange/issues/2 please? Thanks.

**Comment by lukebakken** [comment]
Please see this - https://github.com/rabbitmq/rabbitmq-udp-exchange/issues/2#issuecomment-2007184071

**Reply by tonyg** [reply]
Hi Luke, I totally understand how it could have happened, and many thanks for resolving it so quickly.
